streamlit_demo/README.md

```diff
@@ -0,0 +1,28 @@
+# NNSVS Streamlit demo
+This folder explain how to work with this demo
+
+![demo image](imgs/demo.png)
+
+* install the requirements
+```
+pip install streamlit git+https://github.com/r9y9/nnsvs
+# or if you must specify the python version
+python3.8 -m pip install streamlit git+https://github.com/r9y9/nnsvs
+```
+* download the trained models
+```
+[ ! -e 20220321_kiritan_timelag_mdn_duration_mdn_acoustic_resf0conv ] && curl -q -LO https://www.dropbox.com/s/87rqto5l5rpav2n/20220321_kiritan_timelag_mdn_duration_mdn_acoustic_resf0conv.zip
+unzip -qq -o 20220321_kiritan_timelag_mdn_duration_mdn_acoustic_resf0conv.zip
+
+model_dir = "20220322_yoko_timelag_mdn_duration_mdn_acoustic_resf0conv"
+
+! [ ! -e 20220322_yoko_timelag_mdn_duration_mdn_acoustic_resf0conv ] && curl -q -LO https://www.dropbox.com/s/l1wo9dewfuk3s1v/20220322_yoko_timelag_mdn_duration_mdn_acoustic_resf0conv.zip
+! unzip -qq -o 20220322_yoko_timelag_mdn_duration_mdn_acoustic_resf0conv.zip
+```
```

Comment:
I added the `nnsvs.pretrained` module to manage pretrained modules. Now that you don't need to manually download models in advance. Could you check the updated demo notebook https://github.com/r9y9/nnsvs/blob/master/notebooks/Demos.ipynb and fix the streamlit app accordingly? (it shouldn't take much effort I believe)

The functionality to automatically downloads pretrained models is necessary if we want to deploy this streamlit app to streamlit cloud https://streamlit.io/cloud

Reply:
I've never used huggingface spaces personally. It would be great if we can use it.